### DISCUSSION TITLE
On MT4 EA initializes but on MT5 its stuck on initializing

### DISCUSSION THREAD

**Discussion by decay88** [EA31337/EA31337#402] [General]
Ive backtested the EA and that all works fine but when i load the EA it doesnt make any trades. Ive even paid for 2 vps thinking it was going to work but its not making any trades. On mt 4 it initializes but on mt5 its stuck on initializing.
I am using ELITE v3 rc1
Any help would be greatly appreciated.
Thanks

**Comment by kenorb** [comment]
Could you share your screenshot how it's stuck (which message is that)?

**Reply by decay88** [reply]
<img src="https://i.ibb.co/fCGv7R8/Screenshot-2024-03-26-053521.png">
<https://ibb.co/BgKNRD4>

**Reply by decay88** [reply]
Even mt4 it says initializing then initialized but it doesnt make any trades?
Ive followed the step by step guide and i cant get it to open positions or trade

**Reply by decay88** [reply]
Thanks for your fast response

**Reply by kenorb** [reply]
Based on logs, looks fine. On 2024.03.25 you've enabled EA, on 2024.03.25 you had disabled automated trading, then enabled it again. There is no error as such. Not sure at this point why it wouldn't trade. Can you run the backtest for the past few weeks, if it's trading fine?

**Reply by decay88** [reply]
backtests works for the last couple weeks to last year march. I havent tried earlier than that. But when i put the lot size higher than 1.3 backtesting doesnt work. Also backtests work up till 24 hours from the time its tested

**Reply by decay88** [reply]
mt4 on expert tab

https://ibb.co/6WJ0YDj

**Reply by kenorb** [reply]
Screenshot doesn't exist, can you re-upload?

**Reply by bobymadoff** [reply]
can you help me for setting ea please ? i'm new into ea bot and i don't know much about it

**Reply by kenorb** [reply]
Can you try Advanced version or v2.013.1 instead, is that works better for you?

**Reply by bobymadoff** [reply]
Yes, that's what I did. However, the lite version seems to be working properly.
Can you give me the ideal settings? Because in optimize there are settings but I don't know where to make the adjustments. 
Thank you for help

**Reply by kenorb** [reply]
Ideally please wait a week or few when v3 is released, it's going to be tested in more details, so then you could try to use it.

**Reply by bobymadoff** [reply]
Ok Perfect 
But Can I contact you by email or Telegram or other?
I need more info . Please 
Thank’s

**Reply by kenorb** [reply]
You can join the channel and discussion's group at https://t.me/EA31337 or PM me at https://t.me/kenorb.
